src/ONE.Models.CSharp/Constants/ConfigurationTypeConstants.cs

```diff
@@ -31,5 +31,11 @@ public static class DataSourceDriver
             public const string Id = "e47b66c7-1942-4f9c-b817-e9225c478ea2";
             public static readonly Guid Guid = new Guid(Id);
         }
+
+        public static class Logbooks
```

Comment:
We have an unfortunate mix of singular and plural in this class.  I think that singular is more appropriate.